modules/core/src/main/scala/dev/profunktor/redis4cats/effect/JRFuture.scala

```diff
@@ -17,78 +17,57 @@
 package dev.profunktor.redis4cats.effect
 
 import cats.effect._
-import cats.effect.implicits._
 import cats.syntax.all._
 import io.lettuce.core.{ ConnectionFuture, RedisFuture }
 import java.util.concurrent._
 
-trait RedisBlocker {
-  def ec: Blocker
-}
-
-object RedisBlocker {
-  def apply(blocker: Blocker): RedisBlocker =
-    new RedisBlocker {
-      def ec: Blocker = blocker
-    }
-}
-
 object JRFuture {
 
   private[redis4cats] type JFuture[A] = CompletionStage[A] with Future[A]
 
-  private[redis4cats] def mkBlocker[F[_]: Sync]: Resource[F, Blocker] =
-    Blocker.fromExecutorService(F.delay(Executors.newFixedThreadPool(1)))
```

Comment:
The use of `Blocker` in this library was mainly because of the underlying single-threaded thread pool, not because of "blocking" per se. This is especially needed for transactions and pipelining but it also works for the rest. 

So all the `F.blocking` are incorrect. We need to keep using this thread pool, maybe differently. After all, `Blocker` was just a wrapper.

Reply:
Yeah, if we need to stick to one thread we'll need to use Async.evalOn

Reply:
I wonder if CE3 needs to keep something like `Blocker` as a higher-level way of working with threadpools than `ExecutionContext`. This is working now with `ExecutionContext`, but releasing the EC resource is currently a no-op and it looks like we'll want to copy the release code from `Blocker`.

Reply:
We were doing that before Blocker was introduced ;)

It could probably be something in `std` or a separate microlibrary. @djspiewak, what do you think? Maybe a general `ExecutionContexts` thing like in @tpolecat's Doobie could be added to std?

Reply:
If there's a nice utility here, I'm down to include it in std. It would probably be JVM-specific though. Microlib might be slightly more appropriate, at least for the time being, but I'm open to arguments either way.

Reply:
I don't think that's justified, to be honest I'm happy that `Blocker` is gone in CE3 :) 

Need a custom `ExecutionContext` for your app? Newtype it and call it a day :smiley: